index.js

```diff
@@ -273,6 +274,9 @@ export default {
           //    '0x0' RLP-encoded transaction of all fields in Transaction (minus version, v, r, s)
           //    '0x1' typed encoding matching MetaMask initial implementation of EIP-712
           rawTx.version = '0x1';
+          rawTx.gasTokenId="0x8bb0";
```

Comment:
this is not needed because of line 251

Reply:
but  in  npm  ,  it  is not used  
![image](https://user-images.githubusercontent.com/19581409/65413323-cb725480-de23-11e9-9c20-65d3a4597f6a.png)

Reply:
update the version to 2.0.0
https://cdn.jsdelivr.net/npm/quarkchain-web3@2.0.0
https://www.npmjs.com/package/quarkchain-web3